src/AutomaticVersionRanges.targets

```diff
@@ -0,0 +1,42 @@
+<Project>
+
+  <PropertyGroup>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(Configuration)' == 'Debug'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(IsPackable)' == 'false'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(ManagePackageVersionsCentrally)' == 'true'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == ''">true</AutomaticVersionRangesEnabled>
+  </PropertyGroup>
```

Comment:
**Question:** As I understand it, the new setting will be enabled by default. How we will disable it? 
It will be enough to add `<AutomaticVersionRangesEnabled>true</AutomaticVersionRangesEnabled>` to the `Custom.Build.props` file on the repo?

Reply:
It needs to be set to `false`, not `true`, but otherwise, yes.

Reply:
Yeah, sorry about that!